references/validationFunctions/treatment/clinicalTrialValidation.js

```diff
@@ -48,7 +48,10 @@ const validation = () =>
             result = {valid: false, message: `The submitted EudraCT clinical trial number is in incorrect format.`};
           }
           else if (!clinical_dbs.includes(clinicalTrialsDB)) {
-            result = {valid: false, message: `The submitted clinical trials database '${$row.clinical_trials_database}' is not included in the list of clinical trial database.`};
+            result = {valid: false, message: `The submitted clinical trials database '${$row.clinical_trials_database}' is not included in the list of clinical trial database. To include a new clinical trials database in the controlled terminology for 'clinical_trials_database', please contact the DCC.`};
```

Comment:
like in the genetic_disorders field..you could say
`please contact us at https://platform.icgc-argo.org/contact.`